shell/src/main/java/alluxio/cli/validation/HdfsValidationTask.java

```diff
@@ -85,7 +85,7 @@ private boolean validateHdfsSettingParity(Map<String, String> optionsMap) {
     // If ServerConfiguration does not contain the key, then a {@link RuntimeException} will be
     // thrown before calling the {@link String#split} method.
     String[] clientHadoopConfFilePaths =
-        ServerConfiguration.get(PropertyKey.UNDERFS_HDFS_CONFIGURATION).split(":");
+        InstancedConfiguration.defaults().get(PropertyKey.UNDERFS_HDFS_CONFIGURATION).split(":");
```

Comment:
same here. Create a single `AlluxioConfiguration` object and pass the references around.

`AlluxioConfiguration` should be used rather than `InstancedConfiguration` because `InstancedConfiguration is not  immutable.